src/app/ReadHandler.cpp

```diff
@@ -87,8 +136,21 @@ ReadHandler::~ReadHandler()
     InteractionModelEngine::GetInstance()->ReleaseDataVersionFilterList(mpDataVersionFilterList);
 }
 
-void ReadHandler::Close()
+void ReadHandler::Close(bool keepPersisted)
```

Comment:
`Close(true)/Close(false)` is not intuitive when reading code. Please create an `enum class` with values like `kKeepPersistedSubscriptions` or similar. Then I can read code like `Close(kDropPersistentSubscriptions` and `Close(PersistedSubscriptions::kKeep)` or similar.

Reply:
Thank you for the suggestion.